src/components/Header/NavbarLarge.tsx

```diff
@@ -38,7 +36,7 @@ export const NavbarLarge = ({ navItems }: NavbarLargeProps) => {
     >
       <Flex
         px="1rem"
-        w={{ base: "100%", lg: "95%", xl: "90%" }}
```

Comment:
This is a minor comment, but can you configure your vs code settings to use single quotes instead of double quotes? Otherwise, there will be lots of conflicts and these are not supposed to flag as part of your changes.